app/services/teaching_record/rest_client.rb

```diff
@@ -13,7 +13,7 @@ class TeacherNotFoundError < StandardError; end
     class HttpError < StandardError; end
 
     def self.get(path)
-      response = Request.get("/v3/#{path}")
+      response = Request.get("/v3/#{path.delete(" ")}")
```

Comment:
Can we use `strip` instead of `delete(" ")`? https://apidock.com/ruby/String/strip§

Reply:
I believe the space was actually in the middle of the path in this instance, given there should be no spaces in a path I opted to remove them all.